src/commands/a32nx/weightBalance.ts

```diff
@@ -10,7 +10,7 @@ export const weightBalance: CommandDefinition = {
         const weightBalanceEmbed = makeEmbed({
             title: 'FlyByWire A32NX | Fuel and Weight',
             description: makeLines([
-                'We have recently improved the A32NX flight model and finalized our weight and balance loading integration available on the development version. Please note:',
+                'We have improved the A32NX flight model and finalized our weight and balance loading integration. Please note:',
                 '',
                 '- We have disabled the MSFS fuel and weights UI to prevent issues.',
                 '',
```

Comment:
I reccomend this looks a bit neater when printed out.

```suggestion
                'We have improved the A32NX flight model and finalized our weight and balance loading integration.',
                '',
                '**Please note:**',
                '- We have disabled the MSFS fuel and weights UI to prevent issues.',
```

Reply:
Could the 'please note' be added as a footer? May look nice.